src/cpp/default-value.cpp

```diff
@@ -107,8 +107,12 @@ int get_default_value(const char* port_name, const char* port_args,
     if(pretty)
     {
         nargs = rtosc_count_printed_arg_vals(pretty);
+#ifdef NDEBUG
```

Comment:
I'd personally leave the `(void)variable;` lines in place unconditionally, but that's a minor note.

Reply:
I thought about this, too, but with `NDEBUG`, we will get notified if `n` should get completely unused in the future, so I'll keep it.